integration/testnet/client.go

```diff
@@ -428,3 +433,27 @@ func (c *Client) GetEventsForBlockIDs(
 
 	return events, nil
 }
+
+func getAccount(ctx context.Context, client *client.Client, address sdk.Address) (*sdk.Account, error) {
+	header, err := client.GetLatestBlockHeader(ctx, true)
+	if err != nil {
+		return nil, fmt.Errorf("could not get latest block header: %w", err)
+	}
+
+	// when this is run against an Access node with indexing enabled, occasionally the indexed height
+	// lags behind the sealed height, especially after first starting up (like in a test).
+	// Retry using the same block until we get the account.
+	for {
```

Comment:
Usually operations like this have an upper time limit so we don't run it indefinitely

Reply:
yea, I left it to the context to manage that. the caller uses a 5 second timeout